tests/integration/cmd_pref_config_test.go

```diff
@@ -262,6 +263,23 @@ var _ = Describe("odo preference and config command tests", func() {
 			Expect(configValue).To(Not(ContainSubstring(("PORT"))))
 			Expect(configValue).To(Not(ContainSubstring(("SECRET_KEY"))))
 		})
+
+		It("should be able to unset an environment variable with foo=bar", func() {
```

Comment:
can we have more complex cases? like setting multiple env vars and then unset multiple env vars ( all, subset )?